tableaudocumentapi/field.py

```diff
@@ -71,6 +65,53 @@ def name(self):
 
         return self.id
 
+    @property
+    def id(self):
+        """ Name of the field as specified in the file, usually surrounded by [ ] """
+        return self._id
+
+    @property
+    def caption(self):
+        """ Name of the field as displayed in Tableau unless an aliases is defined """
+        return self._caption
+
+    @property
+    def alias(self):
+        """ Name of the field as displayed in Tableau if the default name isn't wanted """
+        return self._alias
+
+    @property
+    def datatype(self):
+        """ Type of the field within Tableau (string, integer, etc) """
+        return self._datatype
+
+    @property
+    def role(self):
+        """ Dimension or Measure """
+        return self._role
+
+    @property
+    def is_quantitative(self):
+        return self._type == 'quantitative'
+
+    @property
+    def is_ordinal(self):
+        return self._type == 'ordinal'
```

Comment:
sure, I'll have to determine how to word it to make it obvious.  The best example I can think of is a date series, they define an order, but I'll dig through docs to see how it's explained